indico/modules/events/abstracts/operations.py

```diff
@@ -81,6 +83,28 @@ def delete_abstract(abstract, delete_contrib=False):
                            'Abstract "{}" has been deleted'.format(abstract.title), session.user)
 
 
+def merge_person_links(target_abstract, source_abstract):
+    """Add to `target_abstract` new `AbstractPersonLink`s whose `EventPerson`
```

Comment:
"""
    Add to `target_abstract` new `AbstractPersonLink`s whose `EventPerson`
    exists in the `source_abstract` but is not yet in the `target_abstract`.

    :param target_abstract: The target abstract (to which the links should then be added)
    :param source_abstract: The source abstract
    """

The first line only goes in the `"""` line if it fits in one line and the other lines aren't indented according to [PEP-257](https://www.python.org/dev/peps/pep-0257/)